### DISCUSSION TITLE
Jamulus on Centos

### DISCUSSION THREAD

**Discussion by videobuff** [jamulussoftware/jamulus#1480] [Technical Support & Help] [answered]
I have a small server in a serverpark, running centos 7 soon to be 8.
If i try to compile with :  /usr/lib64/qt4/bin/qmake "CONFIG+=nosound headless" Jamulus.pro
I get the error below. On an Ubuntu server at home it compiles ok

 

```
/usr/local/jamulus/Jamulus.pro:364:Unknown replace function: absolute_path
WARNING: target.path is not defined: install target not created
```

I have searched for solutions but cannot find it. Any help is much appreciated.

Erik

**Comment by ghost** [comment, marked as answer]
You must use qt5.
BTW I build Jamulus on CentOS 6

**Reply by videobuff** [reply]
Thanks very much. The error is gone, so 'ill try later all. I will compile the special verson which allow streaming to icecast by dingodoppelt.

**Reply by videobuff** [reply]
David, All is working now on my Centos 7 server very well. It's was little bit seraching for the correct packages and dependencies . The stream I create can be listened to on VLC or webpage with an icecast receiver.  I had to do a little bit of searching for the missing dependencies, but in the end i could find them all and then jamulus compiled fine.
Thanks again.
Erik

**Reply by ghost** [reply]
Streaming Jamulus to VLC is really cool. There are some people here that would be interested (I believe from what I've heard). If you open a new discussion of what you have done, you will get noticed.
